music_assistant/server/providers/jellyfin/__init__.py

```diff
@@ -692,7 +661,7 @@ async def get_stream_details(self, item_id: str) -> StreamDetails:
         if ITEM_KEY_MEDIA_CODEC in media_stream:
             content_type = ContentType.try_parse(media_stream[ITEM_KEY_MEDIA_CODEC])
         else:
-            content_type = ContentType.try_parse(mimetype)
+            content_type = ContentType.try_parse(mimetype) if mimetype else ContentType.UNKNOWN
```

Comment:
If you want, we could also just make the try_parse return Unknown when passing in an invalid mimetype

Reply:
I thought about that but wasn't sure if I liked it or not. It would be stupid to pass an int into that function, so why would I pass None in? But I'm happy to change it if you want.